packages/react-dom/rollup.config.js

```diff
@@ -6,12 +6,12 @@ const { resolve } = createRequire(import.meta.url);
 const reactPkg = new URL("../react/package.json", import.meta.url);
 const reactDomPkg = new URL("./package.json", import.meta.url);
 
-const { name, main } = JSON.parse(fs.readFileSync(reactPkg.pathname, "utf8"));
-const { version } = JSON.parse(fs.readFileSync(reactDomPkg.pathname, "utf8"));
+const { name } = JSON.parse(fs.readFileSync(reactPkg.pathname, "utf8"));
+const { dependencies: { '@esm-bundle/react-dom': version } } = JSON.parse(fs.readFileSync(reactDomPkg.pathname, "utf8"));
```

Comment:
Since React always publishes locked versions of React and React Dom, we can rely on just reading the value of `react-dom` to determine the value of `react`

Reply:
Ack